metadata-ingestion/src/datahub/ingestion/source/sql/snowflake.py

```diff
@@ -7,6 +7,13 @@
 
 # This import verifies that the dependencies are available.
 import snowflake.sqlalchemy  # noqa: F401
+from cryptography.hazmat.backends import default_backend
```

Comment:
I think this would require adding `cryptography` as a dependency for the `snowflake` and `snowflake-usage` plugin in `setup.py`?

Here's what that looks like today:
```
     "snowflake": sql_common | {"snowflake-sqlalchemy<=1.2.4"},
     "snowflake-usage": sql_common$
     | {"snowflake-sqlalchemy<=1.2.4", "more-itertools>=8.12.0"},
```

You might want to pull out a `snowflake-common` variable and drop in the requirements there.